Include full GenerateContentRequest for countTokens (#169)
google-gemini/generative-ai-dart: pkgs/google_generative_ai/CHANGELOG.md

  replies with more than one part.
- Fix handling of `format` argument to `Schema.number` and `Schema.integer`.
- Export `UsageMetadata`.
- Include the full `GenerateContentRequest` (previously omitted
  `safetySettings`, `generationConfig`, `tools`, `toolConfig`, and
  `systemInstruction`) in `countTokens` requests. This aligns the token count

>   with token count the backend will see in practice if it was a

Reviewer: ```suggestion
  with the token count the backend will see in practice for a
```
